modules/cluster-logging-deploy-cli.adoc

```diff
@@ -386,7 +386,7 @@ This creates the {logging} components, the `Elasticsearch` custom resource and c
 
 . Verify the installation by listing the pods in the *openshift-logging* project.
 +
-You should see several pods for OpenShift Logging, Elasticsearch, Fluentd, and Kibana similar to the following list:
+You should see several pods for OpenShift Logging, Elasticsearch, the collector, and Kibana similar to the following list:
```

Comment:
I think we should rephrase it as 'Ensure that Logging EFK components are up and running similar to the following list'